build.gradle

```diff
@@ -64,6 +69,8 @@ dependencies {
   
   implementation 'org.springframework:spring-web:5.2.7.RELEASE'
 
+  implementation 'com.opencsv:opencsv:5.2'
```

Comment:
Is this used anywhere? I did a search for `opencsv` but only found this line.

Reply:
Good catch, will fix.